ajax/libs/aes-js/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "aes-js",
+  "version": "3.0.0",
+  "description": "A pure JavaScript implementation of the AES block cipher and all common modes of operation.",
+  "filename": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ricmoo/aes-js.git"
+  },
+  "keywords": [
+    "aes",
+    "aes-ctr",
+    "aes-ofb",
+    "aes-ecb",
+    "aes-cbc",
+    "aes-cfb",
+    "encrypt",
+    "decrypt",
+    "block",
+    "cipher"
+  ],
+  "author": "Richard Moore <me@ricmoo.com>",
+  "license": "MIT",
+  "npmName": "aes-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

Comment:
The file structure doesn't have a `dist` folder, so please change to the following 
```
"basePath": "",
"files": [
  "index.js"
]
```